backend/package.json

```diff
@@ -74,4 +83,4 @@
     "standard": "^10.0.3",
     "supertest": "^3.0.0"
   }
-}
+}
```

Comment:
nit: every other file ends in a newline (UNIX style) ... we should also end this file in a newline